webrtc.html

```diff
@@ -5186,7 +5186,9 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
           internal slot, representing a list of
           <code><a>MediaStream</a></code> objects that the
           <code><a>MediaStreamTrack</a></code> object of this sender is
-          associated with.</p>
+          associated with. The <a>[[\AssociatedMediaStreams]]</a> slot is used
+          when <var>sender</var> is represented in SDP as described in <span
+          data-jsep="initial-offers">[[!JSEP]]</span>.</p>
```

Comment:
I don't think JSEP needs an internal slot though; it can rely on prose. So I'd recommend just removing `[[AssociatedMediaStreams]]` completely.